addons/account/tests/test_account_move_out_refund.py

```diff
@@ -838,7 +838,10 @@ def test_out_refund_line_onchange_sequence_number_1(self):
 
         self.assertRecordValues(self.invoice, [{'name': 'RINV/2019/0042'}])
 
-        invoice_copy = self.invoice.copy()
+        values = {
```

Comment:
You probably need to make the same fix in `addons/account/tests/test_account_move_in_refund.py `